time_difference.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |gem|
   gem.version       = "0.7.0"
   gem.license = 'MIT'
 
-  gem.add_runtime_dependency('activesupport', '~> 5.1')
+  gem.add_runtime_dependency('activesupport')
```

Comment:
It's better to do something like `'activesupport', '>= 5.1', '< 7'`, because there is no warranty that this gem will work with 7th version, or 8th, and so on.

Reply:
updated